### PR TITLE
Implement Navigation in Changelog

### PR DESCRIPTION
**Pull request by Heyrows**
## Description

Implementing the Navigation in the newly reworked changelog page.

### CODE REVIEW

File: packages/slice-machine/scripts/start/infoxBox.ts [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
 
   console.log(
     boxen(
       dedent(`🍕 SliceMachine ${
-        npmCompare.currentVersion.split("-")[0]
+        packageChangelog.currentVersion.split("-")[0]
       } started.
         ${
-          hasUpdate && npmCompare.onlinePackage
+          hasUpdate && packageChangelog.latestNonBreakingVersion
```
Comment (Heyrows) [inline packages/slice-machine/scripts/start/infoxBox.ts]: I think you meant to use isUpdateAvailable

File: packages/slice-machine/scripts/start/infoxBox.ts [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
             ? `
-          A new version (${npmCompare.onlinePackage.version}) is available!
+          A new version (${packageChangelog.latestNonBreakingVersion}) is available!
           Upgrade now: yarn add slice-machine-ui@latest
```
Comment (Heyrows) [inline packages/slice-machine/scripts/start/infoxBox.ts]: If we use `latestNonBreakingVersion`, the command with latest might break things.

Comment (Heyrows) [reply packages/slice-machine/scripts/start/infoxBox.ts]: OPT: do we want to use yarn or npm ?

File: packages/slice-machine/components/Changelog/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      sx={{
+        maxWidth: "1224px",
+      }}
+    >
+      <Navigation
+        changelog={changelog}
+        selectedVersion={selectedVersion}
+        selectVersion={(version) => setSelectedVersion(version)}
+      />
+      {selectedVersion ? (
```
Comment (Miniplop) [inline packages/slice-machine/components/Changelog/index.tsx]: we can simplify this with selectedVersion && ...

File: packages/slice-machine/components/Changelog/versionDetails/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+          gap: "24px",
+        }}
+      >
+        <UpdateCommandBox
+          changelog={changelog}
+          selectedVersion={selectedVersion}
+          packageManager={packageManager}
+        />
+
+        {selectedVersion.releaseNote ? (
```
Comment (Miniplop) [inline packages/slice-machine/components/Changelog/versionDetails/index.tsx]: we can simplify this line : selectedVersion.releaseNote && ...

**Pull request merged**